src/lib/role/user-role-management.ts

```
@@ -0,0 +1,252 @@
+ */
+export async function removeRoleFromUser(
+  userId: string,
+  tenantId: string,
+  roleId: string
+): Promise<boolean> {
+  try {
+    // Check if this is a global role
+    const role = await getGlobalRole(roleId);
+    
+    // For global roles, update the global role users index
+    if (role && role.isGlobal) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)